web/src/message_list.ts

```diff
@@ -353,30 +370,32 @@ export class MessageList {
         $(document).trigger(new $.Event("message_selected.zulip", opts));
     }
 
-    selected_message() {
-        return this.get(this.data.selected_id());
+    selected_message(): Message {
+        const message = this.get(this.data.selected_id());
+        assert(message !== undefined);
+        return message;
```

Comment:
I suspect that this assert is wrong, and this should permit returning `undefined`.

Reply:
Yup, I was able to reproduce. Thanks!